Services/Sessions/WebOSWebAppSession.m

```diff
@@ -206,7 +210,18 @@ - (int)sendSubscription:(ServiceSubscription *)subscription type:(ServiceSubscri
 - (void) handleMediaEvent:(NSDictionary *)payload
 {
     NSString *type = [payload objectForKey:@"type"];
-
+    
+    if(type == nil){
+        NSString *errorMessage = [payload objectForKey:@"error"];
+        if(errorMessage){
+            [_playStateSubscription.failureCalls enumerateObjectsUsingBlock:^(id failure, NSUInteger idx, BOOL *stop)
+             {
+                 FailureBlock mediaFailure = (FailureBlock)failure;
```

Comment:
You can remove this line by changing the block signature to `^(FailureBlock mediaFailure, NSUInteger idx, BOOL *stop)`, since we expect it to be a `FailureBlock` always.